scipy/stats/tests/test_sampling.py

```diff
@@ -311,7 +311,7 @@ def check_discr_samples(rng, pv, mv_ex):
     rvs = rng.rvs(100000)
     # test if the first few moments match
     mv = rvs.mean(), rvs.var()
-    assert_allclose(mv, mv_ex, rtol=1e-3, atol=1e-1)
+    assert_allclose(mv, mv_ex, rtol=1e-2, atol=1e-1)
```

Comment:
Let's lift `atol` and `rtol` to be keyword arguments of `check_discr_samples` with their previous defaults. Then in the new test that needs a more relaxed tolerance, pass `rtol=1e-2`.

Reply:
Done.